### PR TITLE
build: bump logrus and testify

### PR DESCRIPTION
**Pull request by lucacome**
Updates:
- github.com/sirupsen/logrus from `v1.8.1` to `v1.9.0`
- github.com/stretchr/testify from `v1.7.2` to `v1.8.1`

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -56,7 +56,7 @@ require (
 	github.com/yvasiyarov/go-metrics v0.0.0-20140926110328-57bccd1ccd43 // indirect
 	github.com/yvasiyarov/gorelic v0.0.0-20141212073537-a9bba5b9ab50 // indirect
 	github.com/yvasiyarov/newrelic_platform_go v0.0.0-20140908184405-b21fdbd4370f // indirect
-	golang.org/x/net v0.4.0 // indirect
+	golang.org/x/net v0.3.0 // indirect
```
Comment (shizhMSFT) [inline go.mod:59]: @lucacome This dependency is downgraded. Do you know the reason?

Comment (lucacome) [reply go.mod:59]: I've noticed too, but I couldn't figure out why

Comment (shizhMSFT) [reply go.mod:59]: @lucacome Can you run `go mod why golang.org/x/net` to check? BTW, I've merged all your other PRs.

Comment (lucacome) [reply go.mod]: @shizhMSFT I've rebased after all the other merges and now `v0.4.0` is back...

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**